sysutils/libvirt/Portfile

```diff
@@ -109,7 +108,7 @@ configure.cmd       ../${worksrcdir}/configure
 configure.dir       ${workpath}/build
 build.dir           ${configure.dir}
 post-extract {
-    file mkdir ${configure.dir}
+    xinstall -d -o ${macportsuser} ${configure.dir}
```

Comment:
> the creation resulted in a build directory owned by root.

That's https://trac.macports.org/ticket/50918 which will be fixed in the next version of MacPorts.